mlir/test/Dialect/MPI/ops.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: mlir-opt %s | mlir-opt | FileCheck %s
```

Comment:
Mention why you are using mlir-opt twice.

Reply:
I added a comment above, I'm not 100% on if this is the best approach

Reply:
Drive by comment: The generic form is not necessary, as it by definition uses the generic parser.

Reply:
I changed it back to the default round-trip test, thanks for the input @Dinistro!